### PR TITLE
Updated Line Length

### PR DESCRIPTION
**Pull request by Shani-08**
Urls could be longer than 1024.

### CODE REVIEW

**Comment by Jalle19** [comment]
Do you have an example of this?

File: src/PVRIptvData.cpp [hunk 1]
```diff
@@ -327,8 +327,8 @@ bool PVRIptvData::LoadPlayList(void)
   tmpChannel.strTvgLogo     = "";
   tmpChannel.iTvgShift      = 0;
 
-  char szLine[1024];
-  while(stream.getline(szLine, 1024))
+  char szLine[2000];
```
Comment (afedchin) [inline src/PVRIptvData.cpp]: max length of GET request is 4kB, would it make sense to increase to 4096?

Comment (Shani-08) [inline src/PVRIptvData.cpp]: yep, that will do too.

**Comment by afedchin** [comment]
please squash the commits.

**Comment by Shani-08** [comment]
OK i made a mess of it while squashomg the commits. i will do another PR.

**Pull request closed without merging**